lib/aullar/styling.moon

```diff
@@ -157,6 +180,7 @@ define_class {
 
   delete: (offset, count, opts = {}) =>
     @style_buffer\delete offset - 1, count
+    @sub_style_markers\remove_for_range offset, offset + count
```

Comment:
This is not what you want here - this actually shrinks the buffer, but you now only remove some markers and leave others with incorrect offsets. You want to `shrink` the markers.